src/components/contributors.tsx

```diff
@@ -31,6 +31,10 @@ const contributors: GitHubProfileProps[] = [
         userName: 'BernardAddo',
         role: 'Back-End Engineer',
     },
+    {
+        userName: 'nkyarko',
+        role: 'UI/UX designer',
+    },
```

Comment:
Please use the currect github username. Try and run the appplication on your local device to see if everything looks right
https://deploy-preview-17--mentorship-site.netlify.app/